ledger-service/http-json/src/main/scala/com/digitalasset/http/CommandService.scala

```diff
@@ -218,6 +219,7 @@ class CommandService(
       input: CreateAndExerciseCommand[
         lav1.value.Record,
         lav1.value.Value,
+        ContractTypeId.Template.RequiredPkg,
         ContractTypeId.RequiredPkg,
       ]
```

Comment:
Different from the `LAVUnresolved` suggested above, but maybe worthy of similar treatment.